app/services/payout/bitflyer_service.rb

```diff
@@ -1,58 +1,69 @@
 module Payout
-  class BitflyerService < Service
-    def perform
-      # Writes a message if we should skip a publisher or not.
-      # Implemented in Payout::Service
-      return if skip_publisher?
+  class BitflyerService
+    def self.build
+      new(refresher: Bitflyer::Refresher.build, payout_utils_class: Payout::Service)
+    end
+
+    def initialize(refresher:, payout_utils_class:)
+      @refresher = refresher
+      @payout_utils_class = payout_utils_class
+    end
+
+    # Change to call as soon as we refactor the other services
+    def perform(publisher:, payout_report:)
+      payout_utils = @payout_utils_class.new(payout_report: payout_report,
+                                             publisher: publisher,)
```

Comment:
```suggestion
      @payout_utils_class = payout_utils_class.new(payout_report: payout_report,
                                             publisher: publisher,)
    end

    # Change to call as soon as we refactor the other services
    def perform(publisher:, payout_report:)
```

Reply:
This would require passing in the `publisher` and `payout_report` to `build`, which should only be used for non-dynamic utility/third party modules. As the surrounding code is modularized, the utils class will no longer need to be instantiated and those methods can become `self`, effectively turning it into a module of functions.